src/pages/__test__/cart.test.js

```diff
@@ -0,0 +1,855 @@
+                            }
+                        ],
+                        specs: [
+                            { spec_name: 'Collection', spec_value: 'Colossal' },
+                            { spec_name: 'Color', spec_value: 'Beige' },
+                            {
+                                spec_name: 'Dimensions',
+                                spec_value: '61 cm x 61 cm'
+                            }
+                        ],
+                        images: [
+                            {
+                                image: {
+                                    dimensions: { width: 696, height: 900 },
+                                    alt: null,
+                                    copyright: null,
+                                    url: 'https://images.prismic.io/wizeline-academy/310cc973-3b7c-4a0b-adff-83d8606811f0_1.jpeg?auto=compress,format'
+                                }
+                            },
+                            {
+                                image: {
+                                    dimensions: { width: 696, height: 900 },
+                                    alt: null,
+                                    copyright: null,
+                                    url: 'https://images.prismic.io/wizeline-academy/fcdbe85a-58d4-43db-9ca0-cc61ad24527f_2.jpeg?auto=compress,format'
+                                }
+                            },
+                            {
+                                image: {
+                                    dimensions: { width: 696, height: 900 },
+                                    alt: null,
+                                    copyright: null,
+                                    url: 'https://images.prismic.io/wizeline-academy/4d1a2a7a-c23a-463e-9c6a-ae1dd72a440c_3.jpeg?auto=compress,format'
+                                }
+                            },
+                            {
+                                image: {
+                                    dimensions: { width: 696, height: 900 },
+                                    alt: null,
+                                    copyright: null,
+                                    url: 'https://images.prismic.io/wizeline-academy/daf57bdb-242f-4cf1-b19c-862f7799145b_4.jpeg?auto=compress,format'
+                                }
+                            }
+                        ],
+                        stock: 41,
+                        price: 61,
+                        numberOfItems: 1
+                    }
+                ]
+            }
+        });
+        const mockDispatch = jest.fn();
+        render(
+            <AppContext.Provider value={mockDispatch}>
+                <ShoppingCart />
+            </AppContext.Provider>
+        );
+
+        await waitFor(async () => {
+            // eslint-disable-next-line prettier/prettier
+            expect(screen.getByText(/Total: /i)).toBe('"<h2>Total: $ 633</h2>"');
```

Comment:
This test doesn't match the description of the test.

`Validate that you can update the quantity of items for a particular product in the cart. Don’t forget to validate that you don’t exceed the stock units available for the selected product.`